smart-contract/schema/init_msg.json

```diff
@@ -9,16 +9,6 @@
     "vote_duration"
   ],
   "properties": {
-    "admin_weight": {
```

Comment:
are these schema changes unrelated to 1.8 upgrade?

Reply:
these changes were from before but i'm not sure why the json werent updated...

Reply:
https://github.com/provenance-io/digital-currency-consortium/pull/75/files